{{cookiecutter.project_shortname}}/package.json

```diff
@@ -10,8 +10,9 @@
     "build:js-dev": "webpack --mode development",
     "build:js": "webpack --mode production",
     "build:py": "dash-generate-components ./src/lib/components {{ cookiecutter.project_shortname }}",
+    "build:r": "dash-generate-components ./src/lib/components {{ cookiecutter.project_shortname }} --r-prefix '{{ cookiecutter.r_package_prefix }}'",
```

Comment:
As in https://github.com/plotly/dash-core-components/pull/555 let's change this to a single back-end command `build:py_and_r`, also using `-p package-info.json`.